README.md

```diff
@@ -89,7 +89,8 @@ To encrypt during the network transport to S3, use `S3_PROTO=HTTPS` (the default
 
 The `S3_RETRY_COUNT` defines the number of retries for a request after a retryable failure.  The default is 3 retries.
 The `S3_WAIT_TIME_SEC` defines the initial number of seconds between retries.  The default is 2s.  This wait time will double on each successive failure until the `S3_MAX_WAIT_TIME_SEC` is reached.
-The `S3_MAX_WAIT_TIME_SEC` is the maximum wait value during successive failures.  The default is 30s. 
+The `S3_MAX_WAIT_TIME_SEC` is the maximum wait value during successive failures.  The default is 30s.
+The `S3_NON_DATA_TRANSFER_TIMEOUT_SECONDS` defines the timeout value used for S3_complete_multipart_upload and S3_delete_object.  The default is 300s.  (Note:  This has been added because in some cases with very large files these take a long time to complete and the data transfer thresholds do not apply to these.)
```

Comment:
Just curious: What happens if this value is 0? It looks like this is what we had hard-coded before, so I'm assuming it indicates no timeout. Would that be worth mentioning here?

Reply:
Mistake on my part.  Will fix.

Reply:
If the value is 0 it will simply not wait and retries will begin immediately.  This and S3_WAIT_TIME_SEC are different from the others as it isn't a transport type timeout.  These are part of the exponential backoff.

I could state this but I think it is implied.

Reply:
please write any/all of that down.   leave nothing implied - this s3 plugin already has many complex interactions, so making them explicit will be good for everyone.

Reply:
I second this motion

Reply:
This is done.

Reply:
Does a value of 0 for the `S3_NON_DATA_TRANSFER_TIMEOUT_SECONDS` setting have the same meaning as a value of 0 for `S3_MAX_WAIT_TIME_SEC`?